src/core/stdc/stdio.d

```diff
@@ -939,18 +928,18 @@ else version( CRuntime_DigitalMars )
 else version( CRuntime_Microsoft )
 {
   // No unsafe pointer manipulation.
-  extern (D) @trusted
+  @trusted
```

Comment:
Depending on the VC version detected by the compiler we should import the correct msvcxx here.